src/Eccube/Resource/template/default/Shopping/shipping_edit.twig

```diff
@@ -35,7 +35,13 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 {% endblock javascript %}
 
 {% block main %}
-    <h1 class="page-heading">お届け先の追加</h1>
+    <h1 class="page-heading">
+    {% if app.user is not null and is_granted('ROLE_USER') %}
```

Comment:
thank you, detail information. i get a chance to learn.

`if app.user` is only need when ` the page wasn't covered by a security firewall`.
to avoid exception.
for example, error page (error.twig).
http://stackoverflow.com/questions/11869921/symfony2-is-grantedis-authenticated-fully-during-404-error-page-display-cau

`shipping_edit.twig` is "always" covered by a security firewall.
so, we are no need to use `if app.user` check.

and the other twig file use only `{% if is_granted('ROLE_USER') %}`. 
(Cart/index.twig , Shopping/Complete.twig and ..)

so, we want to make it same.
`{% if app.user is not null and is_granted('ROLE_USER') %}` → `{% if is_granted('ROLE_USER') %}`

How do you think?